client/src/components/App.js

```diff
@@ -29,6 +29,7 @@ class App extends Component {
     super(props);
     this.state = {
       connectedWallet: "",
+      ethTransactions: null,
```

Comment:
Instead of `null`, can we use an empty list, `[]` to make it a bit clearer that this'll be an list?